src/index.js

```diff
@@ -59,12 +60,16 @@ function jsonSchemasToGraphqlSchema (schemas, withMutations = true) {
     fields: () => {
       const result = {};
       for (const [name, type] of context.inputs.entries()) {
+        // We must normalize the name here, to match the normalized type names of each schema
+        let normalizedName = normalizeTypeName(name);
+        if (normalizedName.endsWith('In')) normalizedName = normalizedName.slice(0, normalizedName.lastIndexOf('In'));
```

Comment:
What is this special handling of `In` for?

Reply:
The output for each mutation is always `TypeName(input: TypeNameIn): String`.  Except I noticed that when there is a Definition, the output looks like `DefinitionTypeNameIn(input: DefinitionTypeNameIn): String` . This snippet just removes the `In` from the `DefinitionTypeNameIn(...` so that it matches the format of the other mutations.

I wasn't sure where to go with printing the mutations, because aren't they usually more descriptive?  Like `createTypeName` or `updateTypeName` ?

Reply:
Your comment makes sense, but it isn't obvious to me why this change is needed in this PR. That problem seems like it would have been a problem before starting to validate names. Is there a testcase that fails without this change?

Reply:
I just checked the current master branch, and it is also returning the mutations that are named with the suffix `In`, so the changes here did not affect it.  All of the testcases pass regardless, and none of the existing tests have been modified.  Should I take it out of this PR, since it's a separate issue?